bread/layout/components/shell_header.py

```diff
@@ -12,27 +13,21 @@ def __init__(self, platform, company, actions=(), *args, **kwargs):
 
         super().__init__(
             hg.If(
-                hg.F(
-                    lambda c, e: c["request"].user.is_authenticated
-                    and hasattr(c["request"].user, "preferences")
-                    and c["request"].user.preferences.get(
-                        "user_interface__navigation_menu_extended", True
-                    )
-                ),
-                hg.A(
-                    hg.SPAN(platform, _class="bx--header__name--prefix"),
-                    hg.SPAN(company, style="position: absolute; left: 17rem"),
-                    _class="bx--header__name",
-                    style="font-weight: 400",  # override carbon design
-                    href="/",
-                ),
+                HasBreadCookieValue("sidenav-hidden", "true"),
```

Comment:
For example: The client called setBreadCookie("sidenav-hidden", "true") and we can check the according value here.